WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -1338,6 +1347,12 @@ class MySiteViewModel @Inject constructor(
         }
     }
 
+    private fun onBloggingPromptViewAnswersClick(promptId: Int) {
+        bloggingPromptsCardAnalyticsTracker.trackMySiteCardViewAnswersClicked()
+        val tag = BloggingPromptsPostTagProvider.promptIdSearchReaderTag(promptId)
+        _onBloggingPromptsViewAnswers.postValue(Event(tag))
```

Comment:
Question: is this called on the main thread? If so, should we be using `setValue` instead of `postValue`?